pkg/products/rhsso/reconciler.go

```diff
@@ -742,18 +747,26 @@ func syncronizeWithOpenshiftUsers(keycloakUsers []keycloak.KeycloakAPIUser, ctx
 		})
 	}
 
-	openshiftAdminGroup := &usersv1.Group{}
-	err = serverClient.Get(ctx, k8sclient.ObjectKey{Name: "dedicated-admins"}, openshiftAdminGroup)
 	if err != nil && !k8serr.IsNotFound(err) {
 		return nil, err
 	}
-	for _, kcUser := range keycloakUsers {
-		kcUser.ClientRoles = getKeycloakRoles()
+	for index := range keycloakUsers {
+		keycloakUsers[index].ClientRoles = getKeycloakRoles()
 	}
 
 	return keycloakUsers, nil
 }
 
+func isOpenshiftAdmin(kcUser keycloak.KeycloakAPIUser, adminGroup *usersv1.Group) bool {
+	for _, name := range adminGroup.Users {
+		if kcUser.UserName == name {
+			return true
+		}
+	}
+
+	return false
+}
+
```

Comment:
@philbrookes are we currently adding dedicated-admins to the cluster sso? I don't think that is needed right?

Reply:
no, that isn't necessary, @briangallagher are we doing that here?

Reply:
oh, this is probably a hangover from earlier changes, you can remove this admin reconciling stuff from the RHSSO reconciler @briangallagher

Reply:
ok, will do